apps/users/src/app/controllers/users.controller.ts

```diff
@@ -779,7 +779,6 @@ export class UsersController {
     );
     const { user, viewer } = await this._getUserAndViewer(id, req.$credential);
     if (!user) throw CastcleException.USER_OR_PAGE_NOT_FOUND;
-
```

Comment:
Adding a new line may improve the readability.
- before/after `if`
- after multiline declaration `const`/`let`
- before/after `import`